eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration.php

```diff
@@ -618,6 +618,9 @@ private function addImagePlaceholderSection(ArrayNodeDefinition $rootNode)
                             ->variableNode('options')
                                 ->defaultValue([])
                             ->end()
+                            ->booleanNode('verify_binary_data_availability')
+                                ->defaultFalse()
+                            ->end()
```

Comment:
A description of the side-effects of enabling this option would be great here.

https://symfony.com/doc/current/components/config/definition.html#documenting-the-option

```suggestion
                            ->booleanNode('verify_binary_data_availability')
                                ->defaultFalse()
                                ->info('???')
                            ->end()
```

Reply:
Fixed in eddb699.